manifest.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "ROOT: analyzing petabytes of data, scientifically.",
+  "short_name": "ROOT",
+  "description": "ROOT Website",
+  "theme_color": "#346295",
+  "background_color": "#346295",
+  "display": "standalone",
+  "orientation": "portrait",
+  "scope": "/",
+  "start_url": "/",
+  "icons": [
+    {
+      "src": "/assets/images/icon180.png",
```

This one would benefit from `| relative_url`, too, but I don't know how to register a `.json` file to be processed with liquid / by jekyll. Why don't you just try
```suggestion
      "src": {{"/assets/images/icon180.png" | relative_url}},
```